components/online-payments/FeatureShowcase.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+    <div class="container py-5">
+      <div class="d-flex flex-column align-items-center text-center">
+        <h2 class="section__heading">
+          {{ heading }}
+        </h2>
+      </div>
+      <div class="row">
+        <div
+          class="col-md-12"
+          v-for="(feature, index) in features"
+          :key="feature.heading"
+        >
+          <div class="card" :class="{ 'reverse-order': isOdd(index) }">
+            <div class="card-content col-12 col-lg-5">
+              <h3 class="card-title">{{ feature.heading }}</h3>
+              <p class="card-description">{{ feature.description }}</p>
+            </div>
+            <div
+              class="card-image col-12 col-lg-7 d-lg-flex"
+              :class="{ 'justify-content-lg-end  pe-xxl-4': !isOdd(index) }"
+            >
+              <nuxt-img :src="feature.imgUrl" class="img-fluid" alt="..." />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </template>
+  
+  <script>
+  export default {
+    props: {
+      heading: {
+        type: String,
+        required: true,
+      },
+      features: {
+        type: Array,
+        required: true,
+      },
+    },
+    methods: {
+      isOdd(index) {
+        return index % 2 !== 0
+      },
+    },
+  }
+  </script>
+  
+  <style scoped>
+  h2 {
```

Comment:
I think it's a good idea to add css and avoid styling tags directly. 

I know it's scoped but still a better idea is to give it a class.